core/src/main/java/org/apache/carbondata/core/metadata/schema/table/TableSchemaBuilder.java

```diff
@@ -76,6 +78,18 @@ public TableSchemaBuilder blockletSize(int blockletSize) {
     return this;
   }
 
+  public TableSchemaBuilder localDictionaryThreshold(int localDictionaryThreshold) {
+    this.localDictionaryThreshold = String.valueOf(localDictionaryThreshold);
+    return this;
+  }
+
+
+  public TableSchemaBuilder isLocalDictionaryEnabled(boolean isLocalDictionaryEnabled) {
```

Comment:
change the method name to enableLocalDictionary

Reply:
Done